### PR TITLE
new package: libconfig 1.7.2

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/libconfig/sources [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/hyperrealm/libconfig/archive/v1.7.2.tar.gz
```
Comment (dylanaraps) [inline community/libconfig/sources]: This source should remove the need for `autoreconf`.

```
https://hyperrealm.github.io/libconfig/dist/libconfig-1.7.2.tar.gz
```

File: community/libconfig/build [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+export MAKEINFO=true
+
+autoreconf
```
Comment (dylanaraps) [inline community/libconfig/build]: With the source change these should be removable.

File: community/libconfig/build [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+export MAKEINFO=true
```
Comment (dylanaraps) [inline community/libconfig/build]: Why is this needed again?

Comment (a-schaefers) [reply community/libconfig/build]: lol i thought i had fixed all these

**Comment by a-schaefers** [comment]
ready
